ibis/pandas/tests/test_udf.py

```diff
@@ -98,10 +98,9 @@ def a_single_number(**kwargs):
 
 
 @udf.reduction(
-    input_type=[dt.double, dt.Array(dt.double)],
-    output_type=dt.Array(dt.double),
+    input_type=[dt.double], output_type=dt.Array(dt.double),
 )
-def quantiles(series, quantiles):
+def quantiles(series, *, quantiles):
```

Comment:
do we require kwargs only?

Reply:
This is correct. I think this makes the API a little cleaner (what are data/col args and what are configuration args). Basically I want to only allow configuration args as keyword only args.

Reply:
ok right but the actual interface doesn't require that correct? can you also test with positional args (e.g. lke the original), just for completeness. I agree using kwargs only is a good thing. maybe should enforce it.

Reply:
Let me add a few test/doc to clarify what is allowed and not allowed